src/test/java/posters/pageobjects/pages/browsing/HomePage.java

```diff
@@ -90,12 +91,15 @@ public void validateStructure()
     {
         super.validateStructure();
 
+        // validate Sale Banner
+        SaleBanner.validateStructure();
```

Comment:
The idea of the page/component concept is to have every component in the page wherer it's used, so sales banner should now be a field of this class (as is is/was in header.java). 
General rule of thumb: be careful with static ;)